assets/js/index.js

```diff
@@ -61,6 +61,16 @@ const app = new Vue({
 		list: false
 	},
 	methods: {
+		mouseEntered: function (item, e) {
+			if (this.multiSelect.enabled && (e.buttons && e.buttons==1) && e.button == 0) {
```

Comment:
`e.buttons == 1` instead of `e.buttons==1` , seems not to match your codestyle

+ try not to hook everything in brackets, a general preferance in JS is
```
if (!this.multiSelect.enabled || !(e.buttons && e.buttons==1) || e.button !== 0)
    return;
```

And that way save on horizontal space